static/javascripts/app.js

```diff
@@ -206,8 +232,22 @@ function init() {
   loadImpulseResponses();
 }
 
+
+
 function initializeAudioNodes() {
   context = new webkitAudioContext();
+  recordingDest = context.createMediaStreamDestination();
+  mediaRecorder = new MediaRecorder(recordingDest.stream);
+
+  mediaRecorder.ondataavailable = function(evt) {
```

Comment:
I think we should avoid declaring function in the initializer function of all the nodes. It makes it hard to follow, improve or debug.

Reply:
Agreed, will be fixed.